tsfresh/utilities/dataframe_functions.py

```diff
@@ -787,3 +788,25 @@ def _add_id_column(df_chunk):
     df = df.set_index(df.index.get_level_values(-1))
 
     return df
+
+
+def pivot_list(list_of_tuples):
+    """
+    Helper function to turn an iterable of tuples with three entries into a dataframe.
+
+    The input ``list_of_tuples`` needs to be an iterable with tuples containing three
+    entries: (a, b, c).
+    Out of this, a pandas dataframe will be created with all a's as index,
+    all b's as columns and all c's as values.
+
+    It basically does a pd.pivot(first entry, second entry, third entry),
+    but optimized for non-pandas input (= python list of tuples).
+    """
+    return_df_dict = defaultdict(dict)
+    for result_entry in list_of_tuples:
+        # the result_entry is a tuple with the id, the variable and the value
+        # we turn it into a mapping variable -> id -> value
+        chunk_id, variable, value = result_entry
```

Comment:
You could write
```
for chunk_id, variable, value in list_of_tuples:
```

Reply:
That is absolutely correct. Thanks for the comment!